platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/DefaultSignedStateSentinel.java

```diff
@@ -72,7 +73,10 @@ public void checkSignedStates() {
         if (CompareTo.isGreaterThan(objectRecord.getAge(time.now()), maxSignedStateAge)
                 && rateLimiter.requestAndTrigger()) {
             final SignedStateHistory history = objectRecord.getMetadata();
-            logger.error(EXCEPTION.getMarker(), "old signed state detected, memory leak probable.\n{}", history);
+            logger.error(
+                    EXCEPTION.getMarker(),
+                    "Old signed state detected. The most likely causes are either the node has gotten stuck or a memory leak.\n{}",
```

Comment:
Nit, grammar/phrasing.
```suggestion
                    "Old signed state detected. The most likely causes are either that the node has gotten stuck or that there has been a memory leak.\n{}",
```